next.config.js

```diff
@@ -88,6 +88,11 @@ const nextConfig = {
         destination: '/applications?requestAuth=1&createAccount=1',
         permanent: false,
       },
+      {
+        source: '/auditrequest',
+        destination: 'https://airtable.com/appr1nBRRGx2PTJVh/shrpa99vKKW3xafso',
+        permanent: true,
```

Comment:
I think ’false’ is the better option for this.  Even if we have no plans to change it, permanent redirects can be _very_ permanent, and there's no performance consideration to making sure this URL loads faster by skipping the second lookup.

Reply:
Thanks I went back and forth after reading the docs. Pushed the change to false. 👍